presto-product-tests/src/main/java/com/facebook/presto/tests/sqlserver/Select.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.sqlserver;
+
+import com.teradata.tempto.ProductTest;
+import com.teradata.tempto.Requirement;
+import com.teradata.tempto.RequirementsProvider;
+import com.teradata.tempto.configuration.Configuration;
+import com.teradata.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import java.sql.SQLException;
+
+import static com.facebook.presto.tests.TestGroups.SQL_SERVER;
+import static com.facebook.presto.tests.TpchTableResults.PRESTO_NATION_RESULT;
+import static com.facebook.presto.tests.sqlserver.SqlServerTpchTableDefinitions.NATION;
+import static com.facebook.presto.tests.sqlserver.TestConstants.CONNECTOR_NAME;
+import static com.facebook.presto.tests.sqlserver.TestConstants.KEY_SPACE;
+import static com.facebook.presto.tests.utils.QueryExecutors.onPresto;
+import static com.teradata.tempto.Requirements.compose;
+import static com.teradata.tempto.assertions.QueryAssert.Row.row;
+import static com.teradata.tempto.assertions.QueryAssert.assertThat;
+import static com.teradata.tempto.fulfillment.table.TableRequirements.immutableTable;
+import static java.lang.String.format;
+
+public class Select
+        extends ProductTest
+        implements RequirementsProvider
+{
+    @Override
+    public Requirement getRequirements(Configuration configuration)
+    {
+        return compose(
```

Comment:
Do we need a `compose` here?
You didn't use it here: https://github.com/Teradata/presto/pull/503/commits/01090b1f873a3ff73ba983e5e86e0d9d1cc887cc#diff-96992044a8263d5b64ebe0e853fcb3b3R40